solutions/cses-1091.mdx

```diff
@@ -51,3 +56,69 @@ int main(){
 	return 0;
 }
 ```
+
+</CPPSection>
+
+<JavaSection>
+<Warning>
+Due to the way CSES's grader works (allowing 1 second for all problems + being slow), test cases 5 and 8 TLE (time limit exceeded). USACO gives Java test cases 4 seconds, so this isn't representative of what competing with Java will be like. 
+	
+Also, Java doesn't have a defined multiset function, so we have to use an ordered map with some special modifications, using values as a counter of the number of identical sets. This allows us to distinguish between sets, and remove each set once.
```

Comment:
This is also explained in the multisets section of "More operations on ordered sets"